### ISSUE TITLE
Sorting and rolling file name

### ISSUE THREAD

**Issue by Hackerpcs** [deltabeard/ctrmus#28]
It would be nice to be able to sort the files by name/size/date.

Also for files with long file names a "rolling" effect like CTRXplorer would be useful.

**Comment by deltabeard** [comment]
Yes that would be nice. I will consider adding name, size and date ordering soon. Thanks for opening this issue.

Having long file names scroll would be difficult, but would also be doable. I may wait to add this in the graphical version of ctrmus.
